lesson_02/bin/check_jobs.py

```diff
@@ -0,0 +1,42 @@
+import os
+import time
+import requests
+
+
+BASE_DIR = os.environ.get("BASE_DIR")
+
+if not BASE_DIR:
+    print("BASE_DIR environment variable must be set")
+    exit(1)
+
+JOB1_PORT = 8081
+JOB2_PORT = 8082
+
+RAW_DIR = os.path.join(BASE_DIR, "lesson_02", "data", "raw", "sales", "2022-08-09")
+STG_DIR = os.path.join(BASE_DIR, "lesson_02", "data", "stg", "sales", "2022-08-09")
+
+
+def run_job1():
+    print("Starting job1:")
+    resp = requests.post(
+        url=f"http://localhost:{JOB1_PORT}/",
```

Comment:
no need to hard code the host value, host and port value should be added as function parameters

Reply:
fixed